src/main/java/mate/academy/springboot/datajpa/service/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
It's a good practice to use annotation `Component` if it isn't related to the service, controller or repository